tests/ha/ha_cluster_init.pm

```diff
@@ -0,0 +1,29 @@
+use base "hacluster";
+use testapi;
+use lockapi;
+
+sub run() {
+    # M1 belongs to node1, M2 belongs to node2, M3 unlocked
+    type_string "SuSEfirewall2 off\n";
```

Comment:
can you tell in the code with a comment why the firewall needs to be turned off?